src/aurelia-flatpickr.js

```diff
@@ -77,7 +77,7 @@ export class AureliaFlatpickrCustomElement {
         for(let d = 0; d < modelDates.length; d++) {
             let modelDate = modelDates[d];
 
-            if (view.indexOf(modelDate) > -1) {
```

Comment:
In my testing, indexOf wasn't recognising two identical as being equal, which is why I switched to using getTime()